dockers/broad/illumina_iaap_autocall/README.md

```diff
@@ -0,0 +1,35 @@
+# Illumina Iaap CLI
+
+## Quick reference
+
+Copy and paste to pull this image
+
+#### `us.gcr.io/broad-gotc-prod/illumina-iaap-autocall:1.0.2-1.1.0-1625852425`
```

Comment:
should this command be `docker pull us.gcr.io/broad-gotc-prod/illumina-iaap-autocall:1.0.2-1.1.0-1625852425`?